Misc/NEWS.d/next/Core and Builtins/2020-12-02-20-23-31.bpo-42536.Kx3ZOu.rst

```diff
@@ -0,0 +1,4 @@
+Several built-in and standard library types now ensure that their internal
+result tuples are always tracked by the garbage collector. Previously, they
+could have become untracked by a prior garbage collection. Patch by Brandt
+Bucher.
```

Comment:
You may give the list. Copy from your PR description (I didn't check if it's complete):

```
    collections.OrderedDict.items
    dict.items
    enumerate
    functools.reduce
    itertools.product
    itertools.combinations
    itertools.combinations_with_replacement
    itertools.permutations
    itertools.zip_longest
    zip
```